pkg/controller/node/node_controller.go

```diff
@@ -65,6 +65,9 @@ const (
 
 	// schedulerCRName that we're interested in watching.
 	schedulerCRName = "cluster"
+
+	// LayeringEnabledPoolLabel is the label that enables the "layered" workflow path for a pool.
+	LayeringEnabledPoolLabel = "machineconfiguration.openshift.io/layering-enabled"
```

Comment:
This is defined here and in `pkg/operator/sync.go`. Lets put it in `pkg/controller/common/constants.go` since that seems to be where @jkyros originally put it and that's also where openshift/machine-config-operator#3731 PR puts it as well. I can deal with that merge conflict when my PR lands.